cmd/soroban-rpc/internal/test/transaction_test.go

```diff
@@ -11,12 +11,57 @@ import (
 	"github.com/stretchr/testify/assert"
 
 	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/strkey"
 	"github.com/stellar/go/txnbuild"
 	"github.com/stellar/go/xdr"
 
 	"github.com/stellar/soroban-tools/cmd/soroban-rpc/internal/methods"
 )
 
+type AccountInfo struct {
+	ID       string
+	Sequence string
+}
+
+func getAccount(client *jrpc2.Client, address string) (xdr.AccountEntry, error) {
```

Comment:
@paulbellamy this doesn't seem to be used? I think its uses (which make sense) got lost in the last rebase.

I removed the old uses of `getAccount()` at #438 because it broke the tests but the same checks still make sense when using `getLedger()`

Reply:
Yeah good catch. I think I resolved the merge wrong.

Reply:
I don't think this got solved (`getAccount()` is still unused)